frame/vesting/src/benchmarking.rs

```diff
@@ -375,6 +375,49 @@ benchmarks! {
 		);
 	}
 
+	force_remove_vesting_schedule {
+		let l in 0 .. MaxLocksOf::<T>::get() - 1;
+		let s in 0 .. T::MAX_VESTING_SCHEDULES;
+
+		let source: T::AccountId = account("source", 0, SEED);
+		let source_lookup: <T::Lookup as StaticLookup>::Source = T::Lookup::unlookup(source.clone());
+		T::Currency::make_free_balance_be(&source, BalanceOf::<T>::max_value());
+
+		let target: T::AccountId = account("target", 0, SEED);
+		let target_lookup: <T::Lookup as StaticLookup>::Source = T::Lookup::unlookup(target.clone());
+		
+		// Give target existing locks
+		add_locks::<T>(&target, l as u8);
+		
+		let per_block = T::MinVestedTransfer::get();
+		let transfer_amount = per_block.checked_mul(&20u32.into()).unwrap();
+		// 2 x transfer_amount because we will create 3 vesting schedules and remove one
+		let expected_balance = transfer_amount.checked_mul(&2u32.into()).unwrap();
+		// It will remove last vesting schedule
+		let schedule_index = 2;
+
+		let vesting_schedule = VestingInfo::new(
+			transfer_amount,
+			per_block,
+			1u32.into(),
+		);
+		// Add max vesting schedules
+		for _ in 0..T::MAX_VESTING_SCHEDULES {
```

Comment:
```suggestion
		for _ in 0..s {
```
Or can you re-use the `add_vesting_schedules` function?

Reply:
My inital solution was using `add_vesting_schedules` function, but there was some bug that when I use this function then the verify block and removing function is executed multiple times, I discussed this with @shawntabrizi and he said to use this solution that works, because the bug shouldn't be related to this test. 

If you write `s` instead of `T::MAX_VESTING_SCHEDULES`, it will not execute function inside the for loop and will break the test, I will try to figure out why.

Reply:
Probably since the `s` parameter starts at 0. You can let it start at 1 instead, otherwise there is no vesting schedule.

Reply:
I have updated `let s in 2 .. T::MAX_VESTING_SCHEDULES;` to make it work with your suggestion above.